hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/SCMCommonPlacementPolicy.java

```diff
@@ -314,4 +316,14 @@ public ContainerPlacementStatus validateContainerPlacement(
     return new ContainerPlacementStatusDefault(
         (int)currentRackCount, requiredRacks, numRacks);
   }
+
+  /**
+   * Removes the datanode peers from all the existing pipelines for this dn.
+   */
+  public void removePeers(DatanodeDetails dn,
+      List<DatanodeDetails> healthyList) {
+    if (ScmUtils.shouldRemovePeers(conf)) {
```

Comment:
Instead of calling `ScmUtils.shouldRemovePeers(conf)` repeatedly, can this value be stored in an instance variable?